crates/air-lib/interpreter-data/src/trace.rs

```diff
@@ -42,6 +46,13 @@ impl ExecutionTrace {
     pub fn push(&mut self, value: ExecutedState) {
         self.0.push(value);
     }
+
+    pub fn trace_states_count(&self) -> TraceLen {
+        self.0
+            .len()
+            .try_into()
+            .expect("trace_states_count() TraceLen failed.")
```

Comment:
One should clearly state that it is an overflow.

Reply:
Done